docs/apps/lnw/es2k/es2k-lnw-underlay-lag.md

```diff
@@ -0,0 +1,378 @@
+<!--
+
+ Copyright 2024 Intel Corporation.
+
+ SPDX-License-Identifier: Apache-2.0
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at:
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+/
+- -->
```

Comment:
We don't have copyrights for documentation

Reply:
Do we have a policy to this effect? A number of our documents do have copyright/license notices.

I prefer a short-form notice myself:

```xml
<!--
Copyright 2022-2023 Intel Corporation
SPDX-License-Identifier: Apache-2.0
-->
```